lib/ui/platform_dispatcher.dart

```diff
@@ -1315,8 +1328,9 @@ class ViewConfiguration {
     GestureSettings? gestureSettings,
     List<DisplayFeature>? displayFeatures,
   }) {
+    assert(view == null || window == null);
     return ViewConfiguration(
-      window: window ?? this.window,
+      view: view ?? this.view,
```

Comment:
you need to deal with the window here in addition to the view for backwards compatibility reasons:

```dart
window: window ?? this.window,
```

Reply:
I'll get an error that the `assert(window != null || view != null)` cannot be optimized since both the `window` and `view` properties have the same backing property.

Reply:
```dart
assert(window != null || view != null);
return ViewConfiguration(
	view: view ?? window ?? this.view,
);
```

Reply:
Yeah this is also what I would go for.

Reply:
Thanks @dkwingsmt 👍

Reply:
8199e721121231d2e4871a0576628d729209e843 has the changes requested.